api/python/stats.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: default
+title: Place Statistics
+nav_order: 10
+parent: Python
+grand_parent: API
+---
+
+# Get Statistics for Places.
+
+## `get_stats(dcids, stats_var, obs_dates='latest')`
+
+Returns a time series of statistical values for queried places based on the
+[`StatisticalVariable`](https://browser.datacommons.org/kg?dcid=StatisticalVariable) filtered by the specified dates.
+
+**Arguments**
+
+*   `dcids`: A list of dcids of the
+    [`Place`](https://browser.datacommons.org/kg?dcid=Place) to query for.
+
+*   `stats_var`: The dcid of the
+    [`StatisticalVariable`](https://browser.datacommons.org/kg?dcid=StatisticalVariable).
+
+*   `obs_dates`: The dates of [`Observation`](https://browser.datacommons.org/kg?dcid=Observation)s to include. Can be 'latest', 'all', or an list of dates in 'YYYY-MM-DD' format.
```

Comment:
ISO format, "YYYY", "YYYY-MM" or "YYYY-MM-DD" depending on the data observation period